paasta_tools/long_running_service_tools.py

```diff
@@ -107,6 +107,11 @@ def get_healthcheck_mode(self, service_namespace_config):
             raise InvalidHealthcheckMode("Unknown mode: %s" % mode)
         return mode
 
+    def get_bounce_priority(self):
+        """Gives a priority to each service instance which deployd will use to prioritise services
+        higher numbers are higher priority"""
+        return self.config_dict.get('bounce_priority', 0) * -1
```

Comment:
Yeah, it's somehow nasty both ways isn't it. Docs are a good shout though, I'll add something explicit before we ship.